### PR TITLE
Add optional --yes for Messenger force upload attachment

### LINKED ISSUE #119: CLI skip prompt for CI usage
Maybe `--skip-prompt` or `--no-prompt` ?

### PR DESCRIPTION
**Pull request by xxhomey19**
close #119

### CODE REVIEW

File: src/cli/providers/messenger/attachment.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const help = () => {
     ${chalk.dim('Options:')}
 
       --force   Upload all assets and regenerate bottender-lock.json.
+      --yes     Skip prompt confirmation for force upload all assets.
```
Comment (kpman) [inline src/cli/providers/messenger/attachment.js]: Bottender CLI will not force upload when not passing `--force` option, so the description may be misunderstood.

File: bottender-lock.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (kpman) [inline bottender-lock.json]: No need bottender-lock.json file

File: src/cli/providers/messenger/__tests__/uploadAttachment.spec.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  });
+
+  it('be defined', () => {
+    expect(uploadAttachment).toBeDefined();
+  });
+
+  describe('resolve', () => {
+    it('--yes should work', async () => {
+      const ctx = {
+        argv: { forece: true, yes: true },
```
Comment (kpman) [inline src/cli/providers/messenger/__tests__/uploadAttachment.spec.js]: force

File: src/cli/providers/messenger/attachment.js [hunk 4]
```diff
@@ -99,17 +100,19 @@ export async function uploadAttachment(ctx) {
...
-        message: force
-          ? 'Are you sure you want to force upload all assets?'
-          : 'Is it correct for uploading?',
-        name: 'confirm',
-      },
-    ]);
-
-    if (!promptResult.confirm) {
+    const { confirm } = yes
+      ? { confirm: true }
```
Comment (tw0517tw) [inline src/cli/providers/messenger/attachment.js]: I think it's more readable using an `if` statement than a ternary operator here

**Pull request merged**